src/main/java/org/folio/circulation/domain/representations/changeduedate/BatchChangeDueDateRequestValidator.java

```diff
@@ -0,0 +1,66 @@
+package org.folio.circulation.domain.representations.changeduedate;
+
+import static org.folio.circulation.support.JsonPropertyFetcher.getDateTimeProperty;
+import static org.folio.circulation.support.Result.succeeded;
+import static org.folio.circulation.support.ValidationErrorFailure.failedValidation;
+
+import org.folio.circulation.support.Result;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.RoutingContext;
+import java.lang.invoke.MethodHandles;
+import java.util.List;
+import org.apache.commons.lang3.StringUtils;
+import org.joda.time.DateTime;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BatchChangeDueDateRequestValidator {
```

Comment:
I have a sense that this has already been discussed before. Why is this separate to the request class itself?

Reply:
This is in the beginning of this PR in comments

Reply:
Ok. I reckon I might hold a different view on this separation. I’ll resolve this comment.